### PR TITLE
Switch to Github Actions

### LINKED ISSUE #314: [FEA] Migrate CI to github actions
**Is your feature request related to a problem? Please describe.**
Current CI uses Jenkins, which is being phased out in favor of Github actions

### PR DESCRIPTION
**Pull request by dagardner-nv**
Remove bash & git-lfs from `docker/conda/environments/cuda11.5_ci.yml` as the base OS already contains updated versions of these.
Mark benchmark tests as slow, these should have always been marked slow.

Closes #314

### CODE REVIEW

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Build pull request
+
+on:
+  push:
+    branches:
+      - 'pull-request/**'
```
Comment (mdemoret-nv) [inline .github/workflows/pull_request.yml:21]: In SRF we have `      - "branch-*"` as well to build after merging. We need that here, right?

File: .github/workflows/pull_request.yml [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+          fetch-depth: 0
+
+      - name: Check
+        shell: bash
+        run: ./morpheus/ci/scripts/github/checks.sh
+
+  build:
+    name: Build
+    runs-on: [self-hosted, linux, amd64, cpu16]
+    timeout-minutes: 240
```
Comment (mdemoret-nv) [inline .github/workflows/pull_request.yml:67]: This seems really long

File: .github/workflows/pull_request.yml [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+          lfs: false
+          path: 'morpheus'
+
+      - name: Build:linux:x86_64:gcc
+        shell: bash
+        run: ./morpheus/ci/scripts/github/build.sh
+
+  test:
+    name: Test
+    needs: [check, build]
```
Comment (mdemoret-nv) [inline .github/workflows/pull_request.yml:86]: Didnt we remove `check` from SRF?

File: .github/workflows/pull_request.yml [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+    runs-on: [self-hosted, linux, amd64, gpu-v100-495-1]
+    timeout-minutes: 60
+    container:
+      image: rapidsai/ci:cuda11.5.1-ubuntu20.04-py3.8
```
Comment (mdemoret-nv) [inline .github/workflows/pull_request.yml:90]: We added `--cap-add=sys_nice` to SRF. Do we need that here?

**Review by mdemoret-nv — CHANGES_REQUESTED** [review]
A few questions/comments.

File: .github/workflows/pull_request.yml (hunk 2 above)
Comment (dagardner-nv) [reply .github/workflows/pull_request.yml:68]: Good catch, thats a hold-over from when we built cudf ourselves

File: .github/workflows/pull_request.yml (hunk 3 above)
Comment (dagardner-nv) [reply .github/workflows/pull_request.yml:87]: Actually I was wrong, we do skip the check for branch builds. I'll update this.

**Comment by mdemoret-nv** [comment]
@gpucibot merge

**Pull request merged**